minecraft_servers/mythria/manifest.json

```diff
@@ -7,6 +7,8 @@
   ],
   "social": {
     "web": "https://mythria.net",
+    "web_shop": "https://store.mythria.net",
+    "web_support": "https://discord.gg/CvmKExSyQB",

```

Comment:
"web_support" only supports faq/contact websites, not discord invites. For discord invites, use the "discord"-key in the json file.

Reply:
I removed web_support